您好，本 PR 仅改动了 `internal/profiler/runtime/java/retry.go`（3 行），但 `Build, Lint and Vendor` 检查失败。经排查，失败原因与本 PR 无关，是当前 main 上一处既有的 lint 违规：

```
internal/storage/localfile/localfile_test.go:77:29: Error return value of `os.Chmod` is not checked (errcheck)
```

该问题位于 `internal/storage/localfile/localfile_test.go`（本 PR 并未改动此文件）。由于 CI 的 golangci-lint 会扫描整个仓库（`golangci-lint run ./...`），因此从当前 main 切出的 PR 都会命中这一失败。

我可以另外提一个独立的小 PR 修复这一行（例如改为 `_ = os.Chmod(...)`）；修复合并后再把本 PR rebase 到最新 main，CI 即可通过。

--- a/internal/profiler/runtime/java/retry.go
+++ b/internal/profiler/runtime/java/retry.go
@@ -27,8 +27,8 @@ import (
 )
 
 const (
-	// Starts with an initial delay of 10ms, increasing by 10ms with each retry.
-	// Retries up to 1000 times, with a maximum total wait time of approximately 10 seconds.
+	// Retries up to maxRetries times with a fixed retryInterval delay between
+	// attempts, for a maximum total wait time of approximately 10 seconds.
 	maxRetries    = 1000
 	retryInterval = 10 * time.Millisecond
 
@@ -58,7 +58,7 @@ func RetrySampleProfiler(ctx context.Context, pid, dur, freq int, toolPath, outp
 			}
 		}
 
-		log.Infof("PID[%d] sampling attempt %d/%d (delay: %s)", pid, attempt, maxRetries, delay*time.Duration(attempt))
+		log.Infof("PID[%d] sampling attempt %d/%d (delay: %s)", pid, attempt, maxRetries, delay)
 
 		res := sampleFn(ctx, onePid, dur, freq, toolPath, outputFormat)
 		cmdRes := res[0]

